tap_core/src/signed_message.rs

```diff
@@ -19,11 +46,15 @@ pub struct EIP712SignedMessage<M: SolStruct> {
     pub signature: Signature,
 }
 
+/// Unique identifier for a message
+///
+/// The same message may be signed multiple times, but each signature will *NOT*
+/// have a unique hash
```

Comment:
I have trouble understanding that sentence :/

Reply:
Just updated with a better explanation